Modules/DataModel/Sources/PocketCastsDataModel/Public/Model/Podcast+Settings.swift

```diff
@@ -0,0 +1,117 @@
+import PocketCastsUtils
+import Foundation
+
+extension Podcast {
+    public var isEffectsOverridden: Bool {
```

Comment:
Should this be part of this PR?

I don't see any mention of per-podcast playback settings test steps. 🤔

Reply:
This file was moved in its entirety from the main project to the `DataModel` module. Maybe I need to reclassify using `git` somehow so it captures as a move?

Reply:
This file was moved in its entirety. For some reason, the `Podcast+Settings` file from the project was still around so I deleted.